library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/recorder/mapper/MaskAllTextWireframeMapper.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.recorder.mapper
+
+import android.widget.TextView
+
+internal class MaskAllTextWireframeMapper(
+        viewWireframeMapper: ViewWireframeMapper = ViewWireframeMapper()
+) : TextWireframeMapper(viewWireframeMapper) {
+
+    override fun resolveTextValue(textView: TextView): String {
+        return String(CharArray(textView.text.length) { CHARACTER_MASK })
```

Comment:
question: should we keep spaces at least as is? how browser SR does when applying this mask?

Reply:
I am not sure we can do this as it will affect layout. It is kind of a wait and see situations. After some tests on different apps probably we will apply some adjustments.